Mathlib/Topology/Support.lean

```diff
@@ -103,6 +103,10 @@ theorem tsupport_smul_subset_left {M α} [TopologicalSpace X] [Zero M] [Zero α]
   closure_mono <| support_smul_subset_left f g
 #align tsupport_smul_subset_left tsupport_smul_subset_left
 
+theorem tsupport_smul_subset_right {M α} [TopologicalSpace X] [Zero α] [SMulZeroClass M α]
+    (f : X → M) (g : X → α) : (tsupport fun x => f x • g x) ⊆ tsupport g := by
+  apply closure_mono <| (support_smul_subset_right f g)
```

Comment:
```suggestion
    (f : X → M) (g : X → α) : (tsupport fun x => f x • g x) ⊆ tsupport g := 
  closure_mono <| support_smul_subset_right f g
```
`by apply` is useless most of the time. And the braces are not needed, that's the point of `<| `.